core/src/main/java/com/rudderstack/core/RudderUtils.kt

```diff
@@ -54,4 +54,12 @@ object RudderUtils {
         }
     }
 
-}
\ No newline at end of file
+    internal fun getUTF8Length(message: String): Int {
```

Comment:
make this an extension function String.utfLength()

Reply:
Done.